packages/owlbot-bootstrapper/common-container/interfaces.ts

```diff
@@ -49,4 +49,59 @@ export interface CliArgs {
 export interface InterContainerVars {
   branchName: string;
   owlbotYamlPath: string;
+  // API short name (e.g., accessapproval)
+  apiShortName: string;
+  // API pretty or full name (e.g., Access Approval API)
+  apiPrettyName?: string;
+  // Product documentation (e.g., https://cloud.google.com/cloud-provider-access-management/access-approval/docs)
+  apiProductDocumentation?: string;
+  // Client documentation (e.g., https://cloud.google.com/nodejs/docs/reference/access-approval/latest)
+  apiClientDocumentation?: string;
+  // API Issue tracker (e.g., https://issuetracker.google.com/issues/new?component=190865&template=1161103)
+  apiIssueTracker?: string;
+  // API Release level or launch stage (e.g., Beta)
+  apiReleaseLevel?: string;
+  // API ID (as described by 'name' in googleapis/google/*/*/*_v*.yaml)
+  // This is confusing notation as we also have apiId which refers to the unique identifier;
+  // But this is how most .repo-metadata.json files have their configuration.
+  apiId?: string;
+}
+
+export interface ApiFields {
+  // API short name (e.g., accessapproval)
+  apiShortName: string;
+  // API pretty or full name (e.g., Access Approval API)
+  apiPrettyName?: string;
+  // Product documentation (e.g., https://cloud.google.com/cloud-provider-access-management/access-approval/docs)
+  apiProductDocumentation?: string;
+  // Client documentation (e.g., https://cloud.google.com/nodejs/docs/reference/access-approval/latest)
+  apiClientDocumentation?: string;
+  // API Issue tracker (e.g., https://issuetracker.google.com/issues/new?component=190865&template=1161103)
+  apiIssueTracker?: string;
+  // API Release level or launch stage (e.g., Beta)
```

Comment:
+1, this could be an enum type (or simply and inline `'stable' | 'preview'` type)

Reply:
Hi @suztomo, I'm not sure what these values will be, since I don't think these have an enum from where I will be getting them from (first DRIFT, then service_config.proto - see here cl/476957576)